src/pages/billing/Project.js

```diff
@@ -369,11 +365,11 @@ const ProjectDetail = ({ authorizeAndLoadAccounts, billingAccounts, billingProje
             }
           })
         ])])]),
-        CostCard({ title: 'Total spend', amount: (!!totalCost ? totalCost : '$__.__') })
+        CostCard({ title: 'Total spend', amount: (totalCost !== null ? totalCost : '$__.__') })
       ]),
-      costPerWorkspace.numWorkspaces > 0 && div({ style: { gridRowStart: 2, minWidth: '500px' } }, // Set minWidth so chart will shrink on resize
-        [h(HighchartsReact, { highcharts: Highcharts, options: spendChartOptions })]
-      )
+      costPerWorkspace.numWorkspaces > 0 && div({ style: { gridRowStart: 2, minWidth: 500 } }, [ // Set minWidth so chart will shrink on resize
+        h(Suspense, { fallback: null }, [h(LazyChart, { options: spendChartOptions })])
```

Comment:
Because of the lazy load, I have to use a Suspense with a fallback. Typically a spinner would be the fallback, but as we've talked about I'm trying to avoid another spinner. Since there is nothing below on the page at this point, having no fallback looks fine to me.